OpenUpSA/wazimap-ng-ui
# updating active category menu in rich data panel

## Description
updating active category menu in rich data panel when the user scrolls

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-141

## How to test it locally
* expand the rich data panel
* scroll down
* confirm that the active category menu on the left is getting updated as you scroll

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/142210655-b780d71a-f09c-432f-9444-dda6ed6f771a.png)

![image](https://user-images.githubusercontent.com/53019884/142210699-4f08e56a-1f74-41d0-9ae8-9bf94f060b80.png)


## Changelog

### Added

### Updated
* `profile_loader.js` to add a function that will listen to the scroll event and update the menus

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
